ProcessLib/HydroMechanics/LocalDataInitializer.h

```diff
@@ -115,11 +115,11 @@ namespace HydroMechanics
 ///
 /// \attention This is modified version of the ProcessLib::LocalDataInitializer
 /// class which does not include line elements, allows only shapefunction of
-/// order 2, and provides additional template argument DisplacementDim.
+/// order 2.
```

Comment:
If I remember correctly, the additional template argument was one reason why you can't use the general LocalDataInitializer class. Another reason was, that HM cannot be instantiated for 1D and 0D elements.
Maybe now it's easier to make HM use the general LocalDataInitializer.
I suggest that we put that with a rather low priority somewhere on some agenda.